src/components/pages/search/selectors/selectTypes.js

```diff
@@ -13,6 +13,8 @@ const selectTypeSublabels = createSelector(
   }
 )
 
+export const selectRawTypes = state => state.data.types
```

Comment:
```suggestion
export const selectTypes = state => state.data.types
```
Et modifies celle d'en dessous en fonction de ce qu'elle fait.

Reply:
:+1: